feat: update board and comment functionality

cakefeelsgood opened the pull request.

cakefeelsgood committed:
feat:update board and comment functionality

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/config directory

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/controller/PloggingController.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/controller/UserController.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/CustomUserDetails.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/JoinDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/UserResponseDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/TotalViewResponseDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/TotalViewAndPloggingListDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/PloggingViewResponseDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/PloggingResponseDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/PloggingRequestDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/PasswordUpdateRequestDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/dto/NameUpdateRequestDto.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/entity/Plogging.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/entity/TotalView.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/entity/User.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/jwt directory

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/repository/PloggingRepository.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/repository/TotalViewRepository.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/repository/UserRepository.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/service/CustomUserDetailsService.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/service/PloggingService.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/service/TotalViewService.java

cakefeelsgood committed:
Delete src/main/java/com/ecovision/ecovision/service/UserService.java

Kyoung-M1N commented on src/main/java/com/ecovision/ecovision/dto/CommentRequestDto.java:
댓글 추가 요청할 때에 보드 id도 필요할 것 같아요 requestDto에 해당 필드 추가해주세요.

Kyoung-M1N commented on src/main/java/com/ecovision/ecovision/dto/CommentResponseDto.java:
responseDto에 boardId는 불필요한 값으로 생각됩니다. 해당 필드 삭제해주세요.

cakefeelsgood committed:
feat: BoardController와 CommentController 예외처리 추가 및 경로 수정

- BoardController: 게시물 전체 조회 및 상세 조회 시 예외처리 추가
- CommentController: 예외처리 추가 및 경로 수정 (경로에 따른 변경 사항 반영)

cakefeelsgood committed:
feat: Add boardId field to CommentRequestDto and update related logic

cakefeelsgood committed:
fix: build.gradle
